File: sub-web-model/src/main/java/org/github/base/service/impl/RoleServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.RoleEntity;
import org.github.base.mapper.IRoleMapper;
import org.github.base.service.IRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-05
 */
@Service
public class RoleServiceImpl extends ServiceImpl<IRoleMapper, RoleEntity> implements IRoleService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/RoleServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.RoleEntity;
import org.github.mysql.blog.base.mapper.IRoleMapper;
import org.github.mysql.blog.base.service.IRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class RoleServiceImpl : ServiceImpl<IRoleMapper, RoleEntity>(), IRoleService
